lib/node_modules/@stdlib/utils/parse-ndjson/README.md

```diff
@@ -0,0 +1,158 @@
+<!--
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Parse NDJSON
+
+> Parse a string containing serialized newline-delimited JSON (NDJSON).
+
+## Usage
+
+```javascript
+var parseNDJSON = require( '@stdlib/utils/parse-ndjson' );
+```
+
+### **parseNDJSON( str[, reviver] )**
+Parses a `string` as `newline-delimited JSON`.
+
+```javascript
+var out = parseNDJSON( '{"beep":"boop"}\n{"example":42}' );
+// returns [{'beep':'boop'}, {'example':42}]
+```
+
+If unable to parse a `string` as newline-delimited JSON, the function returns an error.
+
+```javascript
+var out = parseNDJSON( 'beep' );
+// returns <SyntaxError>
+```
+
+To transform the `string` being parsed, provide a `reviver`.
+
+```javascript
+const reviver = function (key, value) {
+    if (key === '' || key === 'beep') {
+        return typeof value === 'string' ? value.toUpperCase() : value;
+    }
+    return typeof value === 'number' ? value * 2 : value;
+}
+
+const str = '{"beep":"boop"}\n{"value": 20}\n{"numbers": [1,2,3]}';
+const out = parseNDJSON(str, reviver);
+// returns [{'beep': 'BOOP'}, {'value': 40}, {'numbers': [2,4,6]}]
+```
+
+## Notes
++ In contrast to the native `JSON.parse()`, this implementation parses `string` as `newline-delimited JSON` and returns an array of parsed JSONs.
```

Comment:
Always include empty lines after headings. Also, you seemed to have stripped out the embedded HTML from the README. We use the `<section>` tags in various build processes. See other package READMEs.